contracts/RewardEscrowV2.sol

```diff
@@ -569,19 +549,33 @@ contract RewardEscrowV2 is
         super._mint(_account, entryID);
     }
 
+    function _getVestingEntryStruct(uint256 _entryID)
```

Comment:
nit: `_unpackVestingEntryStruct`

I assume that's what this function is for

Reply:
Yes, although technically it is getting a `VestingEntry` struct rather than the `VestingEntryPacked` struct. However I agree is perhaps more clear.

Reply:
Done.